cli/src/cpp/utils/external_locators_parser.cpp

```diff
@@ -254,14 +259,16 @@ void external_locators_parser(
                                         filename, profile_name, key);
                                     break;
                                 case ExternalLocatorsList::PARTICIPANT_METATRAFFIC_UNICAST:
-                                    // TODO
+                                    qosprof::domain_participant::builtin::metatraffic_external_unicast_locators::
+                                            print_cost(filename, profile_name, key);
                                     break;
                             }
                             break;
                         case CommonCommands::QUERY:
                             // TODO: query kind should be passed in order to select the corresponding library function
                             break;
                         case CommonCommands::SET:
+                            std::string cost = set_every_element ? values[2] : values[0];
```

Comment:
Suggestion here.
We could leverage that we know the indexes to define

`enum SubElementDefaultIdx
{
  DEFAULT_IDX
};` 

`enum SubElementIdx
{
  KIND,
  EXTERNALITY,
  ADDRESS,
  COST,
  MASK,
  PORT
};` 

so that
```suggestion
                            std::string cost = set_every_element ? values[SubElementIdx::COST] : values[SubElementDefaultIdx::DEFAULT_IDX];
```

This could substitute the numeric values

Reply:
I really like this suggestion! Thanks!